app/assets/javascripts/oxalis/view/scalebar.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import * as React from "react";
+import { connect } from "react-redux";
+import { OUTER_BORDER_ORTHO } from "oxalis/constants";
+import type { OxalisState, FlycamType } from "oxalis/store";
+import { calculateZoomLevel, formatZoomLevel } from "oxalis/view/right-menu/dataset_info_tab_view";
+import type { APIDatasetType } from "admin/api_flow_types";
+
+type Props = {|
+  dataset: APIDatasetType,
+  flycam: FlycamType,
+|};
+
+const scalebarWidthPercentage = 0.25;
+
+function Scalebar({ flycam, dataset }: Props) {
+  return (
+    <div
+      style={{
+        position: "absolute",
+        bottom: 4,
+        right: 4,
+        width: `calc(${100 * scalebarWidthPercentage}% - ${2 * OUTER_BORDER_ORTHO}px)`,
+        height: 25,
+        background: "rgba(0, 0, 0, .3)",
+        color: "white",
+        textAlign: "center",
+        fontSize: 12,
+        lineHeight: "14px",
+        boxSizing: "content-box",
+        padding: 4,
+      }}
+    >
+      <div>{formatZoomLevel(calculateZoomLevel(flycam, dataset) * scalebarWidthPercentage)}</div>
+      <div style={{ width: "100%", height: 10, padding: 4, background: "white" }} />
```

Comment:
I'd suggest a color like `rgba(255,255,255,0.6)` here, so the white bars are somewhat transparent and not that eye-catching.

Reply:
Or opt for my alternative design proposal